translations/monero-core_de.ts

```diff
@@ -140,19 +140,19 @@
     <message>
         <location filename="../components/DaemonConsole.qml" line="108"/>
         <source>Close</source>
-        <translation type="unfinished"></translation>
+        <translation>Schließen</translation>
     </message>
     <message>
         <location filename="../components/DaemonConsole.qml" line="119"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehl + Enter (z.B. help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <source>Daemon doesn&apos;t appear to be running</source>
-        <translation type="vanished">Daemon scheint nicht aktiv zu sein</translation>
+        <translation type="vanished">Dämon scheint nicht aktiv zu sein</translation>
```

Comment:
There's a chance Daemon is going to be renamed to Node in english version. So maybe you could use the german word for Node if it suits better.